docs/tools/developer-tools/lab/README.mdx

```diff
@@ -7,6 +7,28 @@ sidebar_position: 80
+### [Stellar Lab](https://lab.stellar.org/?$=network$id=testnet&label=Testnet&horizonUrl=https:////horizon-testnet.stellar.org&rpcUrl=https:////soroban-testnet.stellar.org&passphrase=Test%20SDF%20Network%20/;%20September%202015;;)
 
-A developer tool where you can use and learn about the Stellar network on the Testnet and Pubnet. You can create accounts, construct and submit transactions, and query Horizon and RPC endpoints without needing to know code.
+Stellar Lab is a developer too our new go-to tool for development, experimenting, and testing, as well as exploring APIs developers use to interact with the Stellar network. Whether you're a developer seeking to test transactions, explore RPC methods or Horizon endpoints, or dive deeper into the ecosystem, Stellar Lab provides a modern and user-friendly interface that makes the process smooth and intuitive.
+
+![Lab: Homepage](/assets/lab/lab.png)
+
+### Features of Stellar Lab
+
+- Easily Create Accounts: Create Accounts on Mainnet, Testnet, and Futurenet using a web UI. You can use Friendbot to fund those accounts directly on Lab for Testnet and Futurenet.
+- Access RPC Methods and Horizon Endpoints: Leverage powerful Soroban RPC methods and Stellar Horizon endpoints in a web UI to interact with the Stellar network and obtain crucial data. Try RPC methods to get states from the ledger like accounts, trustlines, contract wasm, and more.
+- Simulate and Submit: Lab supports the ability to use custom RPC providers so that you could simulate transactions, save transactions, and submit transactions directly using Lab. You can also submit transactions using Horizon.
+- Save and Share API Requests: Easily save your requests and transactions for the future or share them with teammates to build faster together XDR ⇔ JSON Support: We have introduced a canonical [XDR to JSON mapping](https://www.npmjs.com/package/@stellar/stellar-xdr-json-web) which is used on Stella Lab (also used in the [Stellar CLI](https://github.com/stellar/stellar-cli). You can see this at work in the Lab where [XDR can be converted to JSON](https://lab.stellar.org/xdr/view).
+- Mobile Friendly: We heard that you use Lab on the mobile, so we optimized the Lab to have a mobile responsive layout.
+
+![Lab XDR to JSON](/assets/lab/xdr-json-lab.png)
+
+These are the features that are available now. There will be more upcoming features to support smart contracts. We look forward to showing you the future of Stellar Lab.
+
+### What About the Old Lab?
+
+In case you need it, the previous version of Stellar Lab is still accessible at [here](https://old-lab.stellar.org). However, it will no longer be actively maintained. We encourage you to explore the new Lab and if you think there is anything that’s missing, please reach out to us Stellar Github.
```

Comment:
```suggestion
In case you need it, the previous version of Stellar Lab is still accessible [here](https://old-lab.stellar.org). However, it will no longer be actively maintained. We encourage you to explore the new Lab and if you think there is anything that’s missing, please reach out to us Stellar Github.
```